front/app/components/Form/Components/Layouts/utils.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  and,
+  isCategorization,
+  JsonSchema,
+  Layout,
+  Tester,
+  UISchemaElement,
+  uiTypeIs,
+} from '@jsonforms/core';
+import { forOwn, isEmpty } from 'lodash-es';
+
+export interface PageType extends Layout {
+  type: 'Page';
+  /**
+   * The label associated with this category layout.
+   */
+  label: string;
+}
+
+export interface PageCategorization extends UISchemaElement {
+  type: 'Categorization';
+  /**
+   * The label of this categorization.
+   */
+  label: string;
+  /**
+   * The child elements of this categorization which are either of type
+   * {@link PageType}.
+   */
+  elements: PageType[];
+}
+
+export const getSanitizedFormData = (data) => {
+  const sanitizedFormData = {};
+  forOwn(data, (value, key) => {
+    sanitizedFormData[key] =
+      value === null || value === '' || value === false ? undefined : value;
```

Comment:
If we had a single selection field with "true" or "false" options, would that data become undefined here?

Reply:
Those would be strings so no? Is there a case you anticipate this being a boolean in our currently supported controls?